rules/windows/file/file_event/file_event_win_remote_cred_dump.yml

```diff
@@ -0,0 +1,28 @@
+title: Remote Credential Dump
+id: 02773bed-83bf-469f-b7ff-e676e7d78bab
+status: experimental
+description: Detects default filenames outputted by execution of CrackMapExec and Impacket-secretsdump against an endpoint.
+references:
+    - https://github.com/Porchetta-Industries/CrackMapExec
+    - https://github.com/SecureAuthCorp/impacket/blob/master/examples/secretsdump.py
+author: SecurityAura
+date: 2022/11/16
+modified: 2022/11/16
+tags:
+    - attack.credential
+    - attack.t1003
+logsource:
+    product: windows
+    category: file_event
+detection:
+    selection:
+        Image|endswith:
+            - 'svchost.exe'
+        CommandLine|contains:
+            - 'RemoteRegistry'
+        TargetFilename|endswith:
+            - '\Windows\System32\????????.tmp'
+    condition: selection
```

Comment:
Hi,
Don't use list with only 1 element in the detection part
```yaml
    selection:
        Image|endswith: 'svchost.exe'
        CommandLine|contains: 'RemoteRegistry'
        TargetFilename|endswith: '\Windows\System32\????????.tmp'
    condition: selection
```

`CommandLine` do not exist in the file_event  category, so rule will nenver trigger.
```yaml
    selection:
        Image|endswith: 'svchost.exe'
        TargetFilename|endswith: '\Windows\System32\????????.tmp'
    condition: selection
```

Reply:
Thanks! It's my 1st rule so I basically copy/pasted an existing one and went from there. I'll know for the next ones!
For `CommandLine` looks like I just learned that for Sysmon File Creation Events (EID 11)... the Image command line is NOT logged. Which is a shame, since for EDRs (at least, Carbon Black Cloud and Defender for Endpoint), this rule with the CommandLine aspect would work. I don't know how accurate this rule will be without the RemoteRegistry aspect now.

Reply:
can be somethink like that
```yaml
    selection:
        Image|endswith: 'svchost.exe'
        TargetFilename|endswith: '\Windows\System32\????????.tmp'
    selection_cmd_exit:
        CommandLine|contains: 'RemoteRegistry'
    selection_cmd_null:
        CommandLine: 'null'
    condition: selection and 1 of selection_cmd_*
```
But need to be test